README.md

```diff
@@ -114,8 +126,20 @@ server {
 
 ```
 
+In addtion modify your `/etc/nginx/nginx.conf`:
+
+  - Change or add `worker_connections` to `20000`
+  - Change or add `worker_processes` to the number of cores on the nginx machine (eg. `4`)
+  - Change or add `keepalive_timeout` to `20`
+  - Change or add `worker_rlimit_nofile` to `65536`
```

Comment:
this should match the nofiles in `ulimit` and connections should go up, cause if you run the tests for 20K users you're going to have more than 20K connections (yes, its 20K connections per CPU, but just to be sure)